### PR TITLE
Remove useless option

### PR DESCRIPTION
**Pull request by csh519**
`--ml_validation` option is now discarded..
So I removed it from CredData.

### CODE REVIEW

File: benchmark/scanner/credsweeper.py [hunk 1]
```diff
@@ -26,8 +27,8 @@ def init_scanner(self) -> None:
     def run_scanner(self) -> None:
         self.init_scanner()
         subprocess.call([
-            "./venv/bin/python", "-m", "credsweeper", "--path", f"{self.cred_data_dir}/data", "--ml_validation",
-            "--save-json", self.output_dir
+            "./venv/bin/python", "-m", "credsweeper", "--path", f"{self.cred_data_dir}/data", "--save-json",
+            self.output_dir
```
Comment (babenek) [inline benchmark/scanner/credsweeper.py:31]:
```suggestion
            "--jobs", "4", self.output_dir
```

**Review by babenek — COMMENTED** [review]
Even the code is changed - maybe speed up a bit ?

File: benchmark/scanner/credsweeper.py (hunk 1 above)
Comment (csh519) [reply benchmark/scanner/credsweeper.py]: Agreed.

**Review by babenek — APPROVED** [review]

File: benchmark/scanner/credsweeper.py [hunk 2]
```diff
@@ -26,8 +27,8 @@ def init_scanner(self) -> None:
     def run_scanner(self) -> None:
         self.init_scanner()
         subprocess.call([
-            "./venv/bin/python", "-m", "credsweeper", "--path", f"{self.cred_data_dir}/data", "--ml_validation",
-            "--save-json", self.output_dir
+            "./venv/bin/python", "-m", "credsweeper", "--path", f"{self.cred_data_dir}/data", "--save-json",
+            "--jobs", "4", self.output_dir
```
Comment (meanrin) [inline benchmark/scanner/credsweeper.py:31]: Probably a wrong order
Should be `self.output_dir` should be wight after `--save-json`

**Review by meanrin — CHANGES_REQUESTED** [review]
Please check is it a wrong order or not

File: benchmark/scanner/credsweeper.py (hunk 2 above)
Comment (csh519) [reply benchmark/scanner/credsweeper.py:31]: Oh.. I'll fix them.

Comment (csh519) [reply benchmark/scanner/credsweeper.py]: Fixed it with [6e284c2](https://github.com/Samsung/CredData/pull/18/commits/6e284c24ba95e275207489c18ef7ecda44f0f3e2).

File: benchmark/scanner/credsweeper.py (hunk 1 above)
Comment (meanrin) [reply benchmark/scanner/credsweeper.py]: @babenek do number of threads influence execution time? As we removed `multiprocessing` in favor of `threading` python would not be able to utilize more cores due to the global interpreter lock

In my experience only 2 cores being used even if 16 `--jobs` specified on 16 core processor

**Review by meanrin — APPROVED** [review]
LGTM

**Review by babenek — APPROVED** [review]
LGTM

**Pull request merged**